src/client.c

```diff
@@ -1016,18 +1016,32 @@ run_recipe_handler (gpointer user_data)
     RecipeData *recipe_data = (RecipeData*)user_data;
     AppData *app_data = recipe_data->app_data;
     const gchar **env = NULL;
+    gchar *command;
 
     // return the xml doc
     xmlBufferPtr buffer = xmlBufferCreate();
     gssize size = xmlNodeDump(buffer, app_data->xml_doc, recipe_data->recipe_node_ptr, 0, 1);
 
+    command = g_strdup_printf ("%s %s -- %s --stdin",
+                               app_data->rsh_cmd,
+                               recipe_data->connect_uri,
+                               app_data->restraint_path);
+
+    if (app_data->restraint_port != 0) {
```

Comment:
Since this is also supported in restraint client, suggest updating documentation for restraint client in commands.rst file.

Reply:
In addition to commands.rst doc file, restraint client cli examples are also listed in using.rst & develop.rst

Reply:
The reference to the option for the client is already in the commands page. And I also added note about the default port for the service in the install page.
The purpose of the option is to allow the service to use the same port across reboots, and it's added to the client in case that some workflow running standalone needs a specific port. Otherwise, for standalone, port is printed out while running. So not sure if it's really needed to added everywhere.

Reply:
I see the update.  I thought it was done for restraintd.  Still getting used to gibhub.com reviews.